### PR TITLE
Create issue template for button requests

### PR DESCRIPTION
**Pull request by LeviSnoot**
Gives users the ability to easily and quickly request new buttons through a GitHub issue form.

### CODE REVIEW

**Comment by LeviSnoot** [comment]
Looks like this in production:

image

File: .github/ISSUE_TEMPLATE/button-request.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Request a button
+description: Request a button to be added to LinkStack (in the "Pre-defined Sites" block).
+title: "[Button request]: "
+labels: ["button", "enhancement"]
+body:
+- type: input
```
Comment (MagicLike) [inline .github/ISSUE_TEMPLATE/button-request.yml:6]: Nice one! I'd like to add a checkbox for the user to check if they have searched other issues:

```
- type: checkboxes
    attributes:
      label: Is the button already requested?
      description: Please search to see if an issue for the button already exists.
      options:
      - label: I have searched the existing issues
        required: true
```

**Review by MagicLike — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/button-request.yml (hunk 1 above)
Comment (LeviSnoot) [reply .github/ISSUE_TEMPLATE/button-request.yml:6]: Totally agree, let's do it!

**Review by MagicLike — APPROVED** [review]
LGTM

**Comment by JulianPrieber** [comment]
We currently manage feature and button requests via discussions.
I've updated the issues page to reflect this more.
chrome_wUOgSLrMTl

With more contributors, I'd definitely see a reason to switch. We'd need to put some work into the tags first.

**Comment by LeviSnoot** [comment]
> We currently manage feature and button requests via discussions. I've updated the issues page to reflect this more. chrome_wUOgSLrMTl
> 
> With more contributors, I'd definitely see a reason to switch. We'd need to put some work into the tags first.

I totally missed that, my bad. I do think the flow is a little better using issues, so I do want to push for a switch there as it would be easier for me to track and maintain requests using this method.

**Pull request closed without merging**